app/templates/macros/toolkit_forms.html

```diff
@@ -0,0 +1,144 @@
+{% macro text(question_content, service_data, errors, question_number=None) -%}
```

Comment:
Are these macros also something that is shared with supplier app?  Or are they different for admin?